docs/pages/includes/database-access/sql-server-local-proxy.mdx

```diff
@@ -1,24 +0,0 @@
-SQL Server clients connect to a Teleport cluster via the local proxy in
```

Comment:
Just double-checking why this partial is being removed

Reply:
I couldn't find any page sourcing it. Seems referenced removed in this change: https://github.com/gravitational/teleport/pull/11801/files